docs/tests.rst

```diff
@@ -167,33 +167,91 @@ Here are a few tips for writing tests:
 .. _Mocha: https://mochajs.org/
 
 
-Smoketests
-==========
+Functional UI Tests
+===================
 
-We can do more comprehensive front-end testing with the smoketests.
-They're located in the ``smoketests/`` directory. See the ``README.rst``
-file in that directory for how to set them up as well as other details.
+We can do more comprehensive front-end testing with the functional UI tests.
+They're located in the ``tests/functional`` directory.
 
+Installing dependencies
+-----------------------
 
-.. _tests-chapter-qa-test-suite:
+Followed the steps in :ref:`the installation docs <hacking-howto-chapter>`,
+including the test dependencies to make sure you have everything you need to
+run the tests. If you're running the tests against a deployed environment then
+there's no need to install anything other than the test dependencies.
 
-The QA test suite
-=================
+Create test users
+-----------------
+
+Some of the tests require logging in as a administrator, and others require
+logging in as a user. To run these tests you will need to create accounts in
+the target environment. If you're running against a local instance of the
+application you can create these users by running the following script::
+
+  $ ./manage.py shell < ./scripts/create_user_and_superuser.py
```

Comment:
Django has a scripts functionality built in that we should use instead of piping to `shell`. It would be `./manage.py runscript create_user_and_superuser`, and `scripts/create_user_and_superuser.py` would need some minor changes. You can use [`scripts/get_media.py`](https://github.com/mozilla/kitsune/blob/80990a688ae66681922d2182dd429b1336d324a9/scripts/get_media.py) as an example.